introduction.md

```diff
@@ -1,3 +1,5 @@
 This is the introduction text for the introduction file.
 
 Ruby
+* Git
+* Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?